src/amo/reducers/versions.js

```diff
@@ -121,6 +117,33 @@ export const getLoadingBySlug = ({ slug, state }: GetBySlugParams): boolean => {
   return Boolean(infoForSlug && infoForSlug.loading);
 };
 
+type VersionInfoType = {|
+  created?: string,
+  filesize?: number,
+|};
+
+type GetVersionInfoParams = {|
+  _findFileForPlatform?: typeof findFileForPlatform,
+  userAgentInfo: UserAgentInfoType,
+  version: AddonVersionType,
+|};
+
+export const getVersionInfo = ({
+  _findFileForPlatform = findFileForPlatform,
+  userAgentInfo,
+  version,
+}: GetVersionInfoParams): VersionInfoType => {
+  const file = _findFileForPlatform({
+    platformFiles: version.platformFiles,
+    userAgentInfo,
+  });
+
+  return {
+    created: file ? file.created : undefined,
```

Comment:
What is this function going to be used for? I guess it's for an upcoming patch?

I'm concerned about when `file` is falsy. I think it should not create an object with empty properties for that case because it could be confusing. Can it return `null` in that case instead of an object with empty properties?

Reply:
Yes, it will be used in an upcoming patch. It's the selector that will be used when rendering a version to retrieve the info needed from the correct file. You're right, it would be better to just return null when `file` is falsey, which I actually realized when working on the next patch. I'll change this.